### ISSUE TITLE
Allow inset xrange to be specified.

### ISSUE THREAD

**Issue by vext01** [softdevteam/warmup_experiment#356]
![inset](https://cloud.githubusercontent.com/assets/604955/24243933/ca9b941c-0fb4-11e7-8f48-7ef69646c7b8.png)

Here it would be nice to have done something like: `--inset-xrange 50,100` to capture the changepoint in the top of that step.

Thanks

**Comment by ltratt** [comment]
Notice this only makes sense if we also scale the y-axis of the inset to the range of the data shown in the inset.

**Comment by vext01** [comment]
Yes. Depends on #357.

**Comment by snim2** [comment]
Fixed in #363
